AWS/legos/aws_release_elastic_ip/aws_release_elastic_ip.py

```diff
@@ -0,0 +1,52 @@
+##
+# Copyright (c) 2023 unSkript, Inc
+# All rights reserved.
+##
+from pydantic import BaseModel, Field
+from typing import Dict
+import pprint
+
+
+class InputSchema(BaseModel):
+    public_ip: str = Field(
+        title='Public IP',
+        description='Public IP address of the Elastic IP to release.')
+    allocation_id: str = Field(

```

Comment:
In the check lego, we are not capturing the `allocation_id`, and here it is a required field. Will this work if we iterate over just region and public_ip?

Reply:
Resolved